npe2/_inspection/_setuputils.py

```diff
@@ -82,6 +82,18 @@ def get_package_dir_info(path: Union[Path, str]) -> PackageInfo:
                 name, _, value = val.partition("=")
                 info.entry_points.append(EntryPoint(name.strip(), value.strip(), group))
 
+    # # check for pyproject.toml
```

Comment:
(This comment isn't quite in the right place).
When I test on `napari-animation`, the package name get's pulled out of `setup.cfg` correctly, but overwritten by `None` in the `setup.py` path.

I don't think the logic really needs to change - just a guard needed on the `info.package_name` update.

Reply:
yeah, thanks.  packages with multiple config files definitely need work.  the guard would definitely be better than the current behavior! :joy:  I also want to do a little more research to learn what setuptools would actually do in the case of, say, all three files.  I believe their priority is 1) pyproject 2) setup.cfg 3) setup.py,  but need to check.  will add guard here

Reply:
added